pkg/cmd/server/origin/admission/chain_builder.go

```diff
@@ -49,7 +49,11 @@ var (
 
 	// kubeAdmissionPlugins gives the in-order default admission chain for kube resources.
 	kubeAdmissionPlugins = []string{
+		"AlwaysAdmit",
```

Comment:
are we aligned with the new hardcoded order in upstream master?

Reply:
> are we aligned with the new hardcoded order in upstream master?

Yeah.  I commented on this pull as it was in progress.

If there is some drift, we'll decide if the order is ok or if we'll update upstream, but we're really close.